src/carbonado/metrics.rs

```diff
@@ -287,9 +315,102 @@ pub async fn update(path: &Path) -> Result<()> {
         .to_string();
     let metadata = path.metadata()?;
     let day_created = metadata.created()?;
+    let day_prior = day_created
+        .checked_sub(Duration::days(1).to_std()?)
+        .expect("day exists");
+    let day_prior = round_datetime_to_day(day_prior.into());
     let day = round_datetime_to_day(day_created.into());
 
+    let first_of_day = if METRICS_DB
+        .open_tree(DB_DAYS)?
+        .contains_key(day.as_bytes())?
+    {
+        debug!("Day already present");
+        false
+    } else {
+        METRICS_DB
+            .open_tree(DB_DAYS)?
+            .insert(day.as_bytes(), &[1])?;
+        true
+    };
+
+    let dir = env::var("CARBONADO_DIR").unwrap_or("/tmp/bitmaskd/carbonado".to_owned());
+    let mut metrics: MetricsData =
+        serde_json::from_str(&fs::read_to_string(format!("{dir}/metrics.json")).await?)?;
```

Comment:
When we run automated tests, no integration test initializes these files, which ends up causing the error.

Could we add a check here to see if the file exists?